patrol_node:
  ros__parameters:
    initial_points:
      - "0.0,0.0,0.0"

    target_points:
      - "2.0,-0.4,0.0"
      - "0.0,3.0,1.57"
      - "-5.0,3.5,3.14"
      - "5.0,-0.5,0.0"
      - "3.0,2.0,0.87"
      - "0.0,0.0,0.0"
    use_sim_time: true
